react/react-dom.d.ts

```diff
@@ -0,0 +1,71 @@
+// Type definitions for React v0.14 (react-dom)
```

Comment:
I think `react-dom` is a completely separate package from React which is already published here https://github.com/borisyankov/DefinitelyTyped/blob/master/react-dom/react-dom.d.ts 

I also suggest changing the name to `react-dom` instead what's here now. Basically I would recommend splitting out all of the modules into separate packages to match NPM.